lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -231,6 +232,20 @@ def process_action(*args)
       ensure
         Datadog::Contrib::Rails::ActionController.finish_processing(payload)
       end
+
+      # rubocop:disable Style/EmptyElse
```

Comment:
Okay, I thought that a `case` block would return the object passed into the `case` statement if none of the branches are hit. Since I wanted it to return `nil`, I thought I needed to be explicit. Turns out its just another one of those weird language things where it does in fact return `nil` if no branches are met. Probably should remove it then.